### PR TITLE
Show either project creation form or project view page

### PR DESCRIPTION
**Pull request by elcorbs**
Checks status and only renders return page/ project summary if the status is set to submitted. Otherwise render null.

### CODE REVIEW

**Comment by faissalMT** [comment]
Don't worry, I will squash merge this.

File: src/App.js [hunk 1]
```diff
@@ -111,26 +111,39 @@ const BackToProjectOverviewButton = props => (
...
+            schema={formSchema}
+            data={formData}
+            submitProject={submitProjectUseCase}
+            updateProject={updateProjectUseCase}
+            generateUISchema={generateDisabledUISchema}
+          />
+        );
+      }
+      if (projectStatus === "Submitted") {
+        return (<div className="col-md-10 col-md-offset-1">
```
Comment (danielburnley) [inline src/App.js]: Would be good to move this to another function - `renderSubmittedProjectPage` or something

File: src/App.test.js [hunk 2]
```diff
@@ -136,131 +146,133 @@ describe("Viewing a project", () => {
...
-
-      let expectedInputValues = [
-        "Meow",
-        "Fluffy balls of friendship",
-        "Beans",
-        ""
-      ];
-      expect(page.getFormInputValues()).toEqual(expectedInputValues);
-    });
+    describe("Status is submitted", () => {
```
Comment (danielburnley) [inline src/App.test.js]: `Project Status is submitted` - because we have return statuses also

Comment (danielburnley) [reply src/App.test.js]: Additionally - is there a test for if the status is draft?

File: src/Components/NewProjectPage/index.js [hunk 3]
```diff
@@ -105,3 +93,13 @@ export default class NewProjectPage extends React.Component {
     );
   }
 }
+/*
```
Comment (danielburnley) [inline src/Components/NewProjectPage/index.js]: :eyes:

File: src/App.test.js (hunk 2 above)
Comment (faissalMT) [reply src/App.test.js]: Line 295 fills that role

**Pull request merged**